lib/twilio-ruby/framework/domain.rb

```diff
@@ -31,6 +31,10 @@ def request(method, uri, params = {}, data = {}, headers = {}, auth = nil, timeo
           timeout
         )
       end
+
+      def as_json(*)
```

Comment:
Should this be `to_json` or `as_json`?

Reply:
It should be `as_json`. 

I tested this by updating the gem, locally, and then point my app to the local install. At that point my application did not get an error when calling `to_json` on  an insights object.

Reply:
Perhaps a suitable unit test would be to ensure that the data returned by this function is as expected, located [here](https://github.com/twilio/twilio-ruby/tree/main/spec/framework).

Reply:
Hi @rhuberdeau,

Just checking back in. Would you like us to take it from here or is this still WIP? Thanks!